docs/build/sdk/v2/create-market-parameters.md

```diff
@@ -0,0 +1,258 @@
+# Market Parameters
+
+Here we document the possible parameters you can pass to market creation and how
+they affect the market.
+
+## Base Parameters
+
+These parameters are needed in all cases, both when creating a market with a
+pool and without.
+
+```ts
+/**
+ * Base parameters for creating a market.
+ */
+export type CreateMarketBaseParams = {
+  /**
+   * The base asset of the market. Can be ZTG or another
```

Comment:
This should end on a full stop.

Reply:
Also, what's "another" here? Seems unclear what the user is allowed to pick here.